openmetadata-ui/src/main/resources/ui/public/locales/en-US/Database/Db2.md

```diff
@@ -3,8 +3,7 @@
 In this section, we provide guides and references to use the Db2 connector.
 
 # Requirements
-<!-- to be updated -->
-You can find further information on the Kafka connector in the [docs](https://docs.open-metadata.org/connectors/database/db2).
+You can find further information on the DB2 connector in the [docs](https://docs.open-metadata.org/connectors/database/db2).
```

Comment:
@OnkarVO7 can we add the minimum permission a user need (access to `information_schema`, etc.) as well as the queries to set those permissions.

Can we also add requirements for Lineage/Usage (if supported here) and profiler/DQ -> https://github.com/open-metadata/OpenMetadata/blob/7f278c67fd5edb369a79697653f0c66ce8a769a7/openmetadata-ui/src/main/resources/ui/public/locales/en-US/Database/Redshift.md